launchable/commands/record/session.py

```diff
@@ -132,6 +141,7 @@ def session(
         "flavors": flavor_dict,
         "isObservation": is_observation,
         "noBuild": is_no_build,
+        "lineage": lineage if lineage else "",
```

Comment:
[question]
Is empty string handled the same as NULL in the backend side?
Is there no chance that a user can set "" lineage?
Is the empty "" lineage validated to fail somewhere?

Reply:
We can set `None` at the cli (Python) side when it's empty. 
So, remove this `if` statement for easy to read.